server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -589,6 +590,27 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
 
     }
 
+    /**
```

Comment:
I don't think we need this code any more?

Reply:
No, we don't! Good catch, thanks 🙏